### DISCUSSION TITLE
Dropdowns in issue templates ignore query strings

### DISCUSSION THREAD

**Discussion by aaubry** [community/community#44983] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Setting a field from the query string in the form `field_name=value`  doesn't work when that field is a dropdown. This works fine for text fields, but for dropdowns that parameter is ignored. It would make sense to have dropdowns work in the same way as the other fields.

**Comment by corubba** [comment]
It's the same for checkboxes, they too can't be set with query parameters. May be related to the fact that only `textarea` and `input` have a `value` attribute to prefill them.

**Comment by sol3uk** [comment]
I have also found this extremely frustrating. Either the user has to populate the dropdown themselves, or I have to auto populate the item as a textbox instead, as these work fine, but for me where I need to provide the user with an automated value, or for them to figure it out themselves, the only other option I have is to list all possible options as text which is extremely ugly for a nice "issue template"
